src/app/modules/registration/pages/ice/ice-thickness/ice-thickness.page.html

```diff
@@ -50,16 +50,15 @@
         <ion-icon color="primary" name="add-circle-outline" slot="start"></ion-icon>
         <ion-label>{{ "REGISTRATION.ICE.ICE_THICKNESS.ADD_ICE_LAYER" | translate }}</ion-label>
       </ion-item>
-      <app-numeric-input
-        [readonly]="true"
-        [(value)]="iceThickness.IceThicknessSum"
-        label="REGISTRATION.ICE.ICE_THICKNESS.ICE_THICKNESS_SUM"
-        [min]="0"
-        [max]="999"
-        [decimalPlaces]="2"
-        [convertRatio]="100"
-        suffix="cm"
-      ></app-numeric-input>
+      <ion-item>
```

Comment:
Dette ble mye bedre enn før, men det ser fortsatt litt ut som en input, i alle fall på mobil. Hvis det virkelig skal se ut som noe som ikke er en input, kanskje den ikke burde vært i en ion-item, eller  brukt color="light" eller noe for å skille fra resten?
Kanskje også https://ionicframework.com/docs/api/item#disabled gjør at den ser mer ikke-input ut?

Reply:
Disabled funka dårlig, da ble tekst og bakgrunn grå. Men color=light ble brukbart, synes jeg. Takk!